README.md

```diff
@@ -1,2 +1,35 @@
-# documentation
-Documentation for users of Jenkins project infrastructure
+# Documentation
+
+This repository's purpose is to keep track of the knowledge around the Jenkins infrastructure project.
+It gathers information for the following areas:
+
+* "Maintenance" section contains document used to prepare service maintenance like systems upgrade.
+* "Meeting" section contains meeting notes
+* "Service" section contains service description
+
+The way contributors can collaborate in this repository depends on either we want "synchronized" or "asynchronized" communication.
+
+In the cased of "synchronized" communication, it's better to use [hackmd.io](https://hackmd.io/team/jenkins-infra) with documents linked in the jenkins-infra channel. Once the collaboration is done, someone needs to push the document into this repository.
+Feel free to request access, our goals is only to avoid spammers.
```

Comment:
```suggestion
Feel free to request access: restriction of the "push to GitHub" action is only to avoid spammers.
```

Reply:
On hackmd.io external contributors only have comment permission

Reply:
OK then, I'm not sure about what do we mean by "Feel free to request access" ? Which access to what?

Reply:
hackmd.io => https://hackmd.io/team/jenkins-infra